src/applications/proxy-rewrite/sass/style-consolidated.scss

```diff
@@ -2799,7 +2799,7 @@
 }
 /* DO NOT REMOVE w/o DS REVIEW: solves !important conflict*/
 /* Fix for team sites, important is necessary because of button */
- div.ts-header-container button.va-button-link, div.ts-footer-container button.va-button-link{
+ div.ts-header-container button.vcl-modal-open, div.ts-footer-container button.va-button-link{
```

Comment:
This style override didn't seem to be specific enough because it was making the gov banner also displaying with white text. Making this update so that it only applies to the modal open button text.

**before**

![Screenshot 2024-09-06 at 11 57 10 AM](https://github.com/user-attachments/assets/57d22353-9190-4466-8100-2c2fa99aab31)

**after**

![Screenshot 2024-09-06 at 12 00 20 PM](https://github.com/user-attachments/assets/3b1b6a68-cf5e-4b74-9538-8f3eefb83557)

Reply:
I didn't notice this issue when I reviewed Ray's PR 😬  which TeamSite was this on?

Reply:
@randimays  I'm checking `http://localhost:3001/?target=https://www.va.gov/health/`. Which one are you trying? Maybe I need to test multiple.

Reply:
That's the same one I use. Just checked `main` and I can see the defect. Nice catch! Thanks for fixing.

Reply:
I didn't see this one either... thanks Jami